code/modules/clothing/head/garlands.dm

```diff
@@ -0,0 +1,43 @@
+/obj/item/clothing/head/garland
+	name = "Floral Garland"
+	desc = "Someone, somewhere, is starving and wearing this. And its defiantly not you."
+	icon_state = "garland"
+	worn_icon_state = "garland"
+
+/obj/item/clothing/head/garland/equipped(mob/user, slot)
+	. = ..()
+	if(slot_flags & slot)
+		var/datum/component/mood/our_mood = user.GetComponent(/datum/component/mood)
+		if(our_mood?.sanity <= SANITY_NEUTRAL)
+			START_PROCESSING(SSobj, src)
```

Comment:
so what if i fall below that sanity when i already have this put on?

Reply:
Good question 
Don't know ultimately 
These only ever got here once when thet got created and I porter them from a branch into the main code

Reply:
Per my comment in Aths first comment;

You'll constantly regain the +0.05 sanity up to the max defined (which is 100). If you go under wearing this, you'll be battling to regain the sanity. Depending on the mood events present, the change will stagnate or lose, because its such a tiny change.